napari/_qt/qt_viewer.py

```diff
@@ -835,16 +835,20 @@ def _open_folder_dialog(self, choose_plugin=False):
         hist = get_open_history()
         dlg.setHistory(hist)
 
-        folder = dlg.getExistingDirectory(
-            self,
-            trans._('Select folder...'),
-            hist[0],  # home dir by default
-            (
-                QFileDialog.DontUseNativeDialog
-                if in_ipython()
-                else QFileDialog.Options()
-            ),
-        )
+        open_kwargs = {
+            "parent": self,
+            "caption": trans._('Select folder...'),
+        }
+        if "pyside" in QFileDialog.__module__.lower():
+            # PySide6
+            open_kwargs["dir"] = hist[0]
+        else:
+            open_kwargs["directory"] = hist[0]
```

Comment:
Why this change is required.

Reply:
I was mimicking the dialog that works above, but I can test keeping/using the positional args instead.